plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/NatureUtils.java

```diff
@@ -26,18 +26,14 @@
 public class NatureUtils {
 
   /**
-   * Returns {@code true} if the project is accessible and has the
-   * specified nature ID.
-   *
-   * @return {@code true} if the project is accessible and has the
-   *         specified nature ID
+   * Returns {@code true} if the project is accessible and has the specified nature ID.
```

Comment:
If we have to pick one, I'd prefer the @return clause for automated tooling purposes.